app/src/main/java/com/example/marvelapp/mvvm/model/CharactersModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.marvelapp.mvvm.model
+
+import com.example.marvelapp.entity.Character
+import com.example.marvelapp.usecase.GetCharactersUseCase
+import com.example.marvelapp.util.Result
+import org.koin.core.component.KoinComponent
+
+class CharactersModel(private val getCharacterUseCase: GetCharactersUseCase) : KoinComponent {
```

Comment:
Extend the KoinComponent here is not necessary

Reply:
fixed